File: repos_estudiantes_ALVAREZ_segunda_entrega/ILLANES_VASQUEZ_GABRIEL/Practica2prog/Composicion y Agregacion/Ejer 9/Producto.java
package Agr9;

public class Producto {
    private String nombre;
    private double precio;

    public Producto() {
        
        this.nombre = "Producto sin nombre";
        this.precio = 0.0;
    }

    public Producto(String nom, double pre) {
        this.nombre = nom;
        this.precio = pre;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public double getPrecio() {
        return precio;
    }

    public void setPrecio(double precio) {
        this.precio = precio;
    }

    public void mostrar_info() {
        System.out.println("Nombre: " + nombre + "  Precio: " + precio);
    }
}

File: repos_estudiantes_TARQUI_segunda_entrega/RUBIN_DE_CELIS_VALDEZ_ANDRES/AGREGACION/EJERCICIO-9/Producto.py
class Producto:
    def __init__(self, nombre: str, precio: float):
        self.__nombre = nombre
        self.__precio = precio

    def get_nombre(self) -> str:
        return self.__nombre
    
    def set_nombre(self, nombre: str) -> None:
        self.__nombre = nombre
    
    def get_precio(self) -> float:
        return self.__precio
    
    def set_precio(self, precio: float) -> None:
        self.__precio = precio
    
    def mostrar_info(self) -> None:
        print(f"Producto: {self.__nombre}, Precio: ${self.__precio:.2f}")
